lesson02/activity/css/_components.content.scss

```diff
@@ -1,13 +1,37 @@
-html {
-    box-sizing: border-box;
-}
+.static-block {
 
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
-}
-
-.main {
-    padding: 2rem;
-}
+    &__title {
+        margin: 4rem 0 0;
+        font-size: 1.5rem;
+        font-family: $font-fam;
+        font-weight: 400;
+        color: $red;
+    }
+    &__copy {
+        margin-bottom: 1rem;
+        font-size: 1rem;
+        font-family: $font-fam;
+        color: $purple;
+    }
+    &__img {
+        display: block;
+        max-width: 100%;
+        margin: auto;
+        border-bottom: 1px solid $blue;
+    }
+    &__content {
+        border-bottom: 1px solid $grey;
+        margin: 2rem auto;
+    }
+    &:only-of-type {
+        .static-block__content {
+            border: none;
+        }
+    }
+    
+    &:last-of-type {
+        .static-block__content {
+            border: none;
+        }
```

Comment:
Here and on line 26, selector can be compressed to a single line, since there are no rules being defined for  `&:last-of-type` itself, only its child: 
` &:last-of-type .static-block__content`